sklearn/utils/extmath.py

```diff
@@ -28,15 +29,15 @@
 from ..exceptions import NonBLASDotWarning
 
 
-def norm(x):
+@deprecated("sklearn.utils.extmath.norm was deprecated in version 0.19"
+            "and will be removed in 0.21. Use scipy.linalg.norm instead.")
+def norm(x, *args, **kwargs):
```

Comment:
Given that the former version only took `x`, I think you could keep it only taking `x``. But I'm okay with this.